src/machines/java/com/enderio/machines/common/blockentity/task/SpawnTask.java

```diff
@@ -137,17 +182,17 @@ public boolean trySpawnEntity(BlockPos pos, ServerLevel level) {
                     }
                 }
 
-                if (entity == null) { //TODO Make default spawn tag?
+                if (entity == null) {
                     blockEntity.setReason(PoweredSpawnerBlockEntity.SpawnerBlockedReason.UNKOWN_MOB);
                     break;
                 }
 
                 if (entity instanceof Mob mob) {
-                    SpawnGroupData res = net.minecraftforge.event.ForgeEventFactory.onFinalizeSpawn(mob, level, level.getCurrentDifficultyAt(pos), MobSpawnType.SPAWNER, null, null);
-                    if (res == null) {
-                        blockEntity.setReason(PoweredSpawnerBlockEntity.SpawnerBlockedReason.OTHER_MOD);
-                        return false;
-                    }
+//                    Event.Result res = ForgeEventFactory.onFinalizeSpawn(mob, level, (float) x, (float) y, (float) z, null, MobSpawnType.SPAWNER);
```

Comment:
Why is this commented? what is the purpose of the commented logic?

Reply:
Oh good catch! That shouldn't be updated, but forge made changes and I didn't get it to compile without removing it.